sklearn/preprocessing/_encoders.py

```diff
@@ -761,20 +786,45 @@ def fit(self, X, y=None):
 
     def transform(self, X):
         """Transform X to ordinal codes.
-
```

Comment:
please do not remove blank lines (or maybe an editor setting that does this automaticall?à

Reply:
Whoops-- I'd love to blame my editor but I think that one was me. Sorry about that.